imports/plugins/included/default-theme/client/styles/variables.less

```diff
@@ -1,6 +1,7 @@
 //== Google Fonts
 //*requires browser-policy to be set in client
 @import url(//fonts.googleapis.com/css?family=Source+Sans+Pro:400,400i,600,700,900);
+@import "/node_modules/react-select/less/select.less";
```

Comment:
This should be put in the `main.less` file. You should be able to put it above the `variables.less` import in that file and get the same effect.